src/gsy_e/models/market/future.py

```diff
@@ -207,7 +207,7 @@ def bid(self, price: float, energy: float, buyer: str, buyer_origin: str,
                                         "method in future markets.")
         bid = super().bid(price=price, energy=energy, buyer=buyer, buyer_origin=buyer_origin,
                           bid_id=bid_id, original_price=original_price,
-                          add_to_history=False, adapt_price_with_fees=adapt_price_with_fees,
+                          add_to_history=True, adapt_price_with_fees=adapt_price_with_fees,
```

Comment:
Not sure, but shouldn't this be:
```suggestion
                          add_to_history= add_to_history, adapt_price_with_fees=adapt_price_with_fees,
```

Reply:
Not really, in the future market case we have to always set it to True because the `offer_history` and `bid_history` is what we use while exporting data

Reply:
We do not want to put all the bids into the `bid_history`. Have a look at the `split_bid` method, the bid that is boing matched, should not be part of the history, as the original bid already is. The residual is set to be `add_to_history=True`. This is why you need to inherit the setting here, as I suggested.

Reply:
Ah I see, you're right, adapted